ui/app/templates/components/secret-edit.hbs

```diff
@@ -12,11 +12,11 @@
     <p.levelLeft>
       <h1 class="title is-3">
         {{#if (eq @mode "create")}}
-          Create secret
+          Create Secret
         {{else if (and this.isV2 (eq @mode "edit"))}}
-          Create new version
+          Create New Version
         {{else if (eq @mode "edit")}}
-          Edit secret
+          Edit Secret
```

Comment:
I'm not sure you can "edit" a secret. You can only create a new version? Something to be addressed in the new KV work.